planning_scene_monitor_options:
  name: "planning_scene_monitor"
  robot_description: "robot_description"
  joint_state_topic: "/joint_states"
  attached_collision_object_topic: "/move_group/planning_scene_monitor"
  publish_planning_scene_topic: "/move_group/publish_planning_scene"
  monitored_planning_scene_topic: "/monitored_planning_scene"
  wait_for_initial_state_timeout: 10.0

planning_pipelines:
  pipeline_names: ["ompl"]

ompl:
  planning_plugin: ompl_interface/OMPLPlanner
  planning_plugins:
    - ompl_interface/OMPLPlanner
  request_adapters:
    - default_planning_request_adapters/ResolveConstraintFrames
    - default_planning_request_adapters/ValidateWorkspaceBounds
    - default_planning_request_adapters/CheckStartStateBounds
    - default_planning_request_adapters/CheckStartStateCollision
  response_adapters:
    - default_planning_response_adapters/AddTimeOptimalParameterization
    - default_planning_response_adapters/ValidateSolution
    - default_planning_response_adapters/DisplayMotionPath

plan_request_params:
  planning_attempts: 10
  planning_pipeline: ompl
  planner_id: "RRTConnectkConfigDefault"
  max_velocity_scaling_factor: 1.0
  max_acceleration_scaling_factor: 1.0

moveit_controller_manager: moveit_simple_controller_manager/MoveItSimpleControllerManager

robot_description_kinematics:
  arm:
    kinematics_solver: kdl_kinematics_plugin/KDLKinematicsPlugin
    kinematics_solver_search_resolution: 0.005
    kinematics_solver_timeout: 0.05
